google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -55,6 +55,24 @@
  *   <li>The default service address (bigtable.googleapis.com) and default port (443) are used.
  *   <li>Credentials are acquired automatically through Application Default Credentials.
  *   <li>Retries are configured for idempotent methods but not for non-idempotent methods.
+ *   <li>These are the default timeouts used Bigtable operations.
```

Comment:
This needs a lot of work and should probably be in its own paragraph and/or in each method doc:
- A distinction should be drawn between idempotent methods (ReadRow(s), MutateRow(s), SampleRowKeys) and non-idempotent methods (CheckAndMutateRows, ReadModiifyWrite)
- The distinction for streaming vs non-streaming should be kept. But there needs to be some explanation about how the settings apply. For example rpc timeout for non-streaming is the deadline for each call, but for streaming the rpc timeout is the timeout between each row. 
- There should be docs explaining what rpc timeout is vs total timeout (ie. attempt timeout vs operation timeout)
- The values should be human readable, ie instead of 20_000ms, it should be 20 seconds
- There should be docs about exponential delay defaults 
- There should be an explanation of what idleTimeout is 
- The formatting is currently invalid: ```<ul>TEXT<li>first element text``` is not valid markup